### ISSUE TITLE
Groepsfotofoto festival eindfeest toevoegen aan header

### ISSUE THREAD

**Issue by jpingen** [lesteenman/pcsa-drupal-theme#43]

**Comment by lesteenman** [comment]
Wacht, is hier een foto van? Die kon ik eerder niet vinden.

Uberhaupt kunnen er denk ik een stuk meer foto's in de header, al zijn er weinig echte groepsfoto's verder. Maar wat andere 'culturele' foto's zijn waarschijnlijk ook wel leuk.

**Comment by jpingen** [comment]
Foto hier: https://lh3.googleusercontent.com/-4HoSGMBq_aA/WA9RUeMZuhI/AAAAAAABW0U/tv19gv76pWwHTiipyc_T1G3bcZn0DciyQCHM/s1600/IMG_0407.jpg

**Comment by jpingen** [comment]
Groepsfotos van Milaan: https://www.facebook.com/media/set/?set=oa.569948869739240&type=3

**Comment by PauluzzS** [comment]
zie ook
https://agoraenschede.org/about/aegee-enschede/primum-collegium-societatis-aegee-enschede-incognito/
Kan ook wel bronbestanden zoeken

Groet,

Paul L. Smits

2017-09-06 20:41 GMT+02:00 Erik <notifications@github.com>:

**Comment by jpingen** [comment]
Er is ook nog ergens een groepsfoto in Thailand bij het legendarische Jim Thompson House.. Kan die alleen niet vinden.
Related: Foto's van Thailand naar de Flickr porten?

**Comment by lesteenman** [comment]
Damn, foto's van Thailand staan nog niet op Flickr? Dat is ook wel een
issue an sich ja.

Moet nog even fixen dat ik ook lokaal gehoste bestanden hiervoor kan
gebruiken. Vanaf Flickr heb ik makkelijk meerdere formaten URLs, waardoor
ik op telefoons geen full-res foto hoef te laden, en we eerst een blurry
preview kunnen laten zien voordat de hele foto geladen is.

**Comment by lesteenman** [comment]
Note: Hart is gemaild over de foto's voor Thailand.

**Comment by jpingen** [comment]
Foto festival eindfeest bijknippen, nu vallen precies alle hoofden eraf 

![image](https://user-images.githubusercontent.com/16253465/30205439-07514768-9489-11e7-8805-8a94e306a811.png)

**Comment by lesteenman** [comment]
Meh, is afhankelijk van je schermbreedte ;)

Maar kan wel, dan knip ik de aegee banner er vanaf op Flickr.

**Comment by lesteenman** [comment]
Er staan nu 7 foto's in. Wanneer we meer leuks tegen komen gooi ik ze er bij, maar voor nu is dit wel een aardige hoeveelheid.

**Comment by Maharball1** [comment]
Ja, we moeten zorgen dat er vooral eerst een aantal foto's van actieven te zien is, en dan bedoel ik vooral Westerbeek, de Regt, Tenger, Lohman en Holst. Sjaar zullen zich daar het eerst mee afficheren en die ouwe lullen zijn dan mooi om te zien. Alleen ouwe lullen lijkt het toch een site van onbekende mensen. Ik zal eens kijken of ik ergens wat leuke foto's heb rondzwerven, al mogen dat wat mij betreft ook foto's in kleinere groepen zijn.

**Comment by jpingen** [comment]
Die UT nieuws groepsfoto is wel mooi!

**Comment by jpingen** [comment]
Related: volgens mij zijn de header images somewhat stuk. Ze laden heel traag en af en toe helemaal niet.
